Kickstarter-iOS/Views/Controllers/PledgeAmountSummaryViewController.swift

```diff
@@ -0,0 +1,128 @@
+import KsApi
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class PledgeAmountSummaryViewController: UIViewController {
```

Comment:
This is just copy-pasting the required views from `ManagePledgeSummaryView`.

Reply:
As discussed on Zoom, I'll create a follow-up PR to incorporate this as as subview of `ManagePledgeSummaryViewController` to promote reuse.